dev-docs/bidders/jwplayer.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: bidder
+title: JW Player
+description: Prebid JW Player Bidder Adapter
+biddercode: jwplayer
+gvl_id: 1046
+pbjs: true
+pbs: false
+media_types: video
+userIds: all (with commercial activation)
+tcfeu_supported: true
+dsa_supported: true
+prebid_member: true
+schain_supported: true
+coppa_supported: true
+usp_supported: true
+gpp_supported: true
+floors_supported: true
+fpd_supported: true
+deals_supported: true
+prebid_member: yes
+sidebarType: 1
+---
+
+<a id="table-of-contents"></a>
+
+## Table of contents
+
+* [Table of contents](#table-of-contents)
+* [Introduction](#introduction)
+* [Modules to include in your build process](#modules-to-include-in-your-build-process)
+* [Bid Params](#bid-request-parameters)
+  * [Required Bidder Params](#required-bidder-params)
+  * [mediaTypes.video](#mediatypes-video)
+  * [First Party Data (FPD)](#set-up-first-party-data-fpd)
+* [Examples](#examples)
+
+<a id="introduction"></a>
+
+## Introduction
+
+The JW Player Bid Adapter allows publishers to tap into JW Player's Video Advertising Demand.
+Publishers are not required to use JW Player as a video player.
+Instream and outstream video ads are supported.
+
+<a id="modules-to-include-in-your-build-process"></a>
+
+## Modules to include in your build process
+
+You will need to include the `jwplayerBidAdapter` in your build. If you are building the JS binary on your own from source code, follow the instructions in [Prebid.js project README](https://github.com/prebid/Prebid.js/blob/master/README.md#build-optimization). 
+We recommend including the [jwplayerVideoProvider](dev-docs/modules/jwplayerVideoProvider.md) to connect Prebid.js to your JW Player instance via the [Prebid Video Module](prebid-video/video-module.md). 
+If you are not using the JW Player Video Provider, we suggest including the JW Player Real Time Data Provider [jwplayerRtdProvider](dev-docs/modules/jwplayerRtdProvider.md) in order to include JW Player's contextual ad targeting segments in your bid requests. 
+
+The following is an example build command that include these modules: <br />
+`gulp build --modules=jwplayerBidAdapter,jwplayerVideoProvider`<br />
+or <br />
+`gulp build --modules=jwplayerBidAdapter,jwplayerRtdProvider`
+
+If you are using a JSON file to specify modules, add `jwplayerBidAdapter`, `jwplayerVideoProvider` and `jwplayerRtdProvider` to the modules array as follows:
+
+```javascript
+[
+    "jwplayerBidAdapter",
+    "jwplayerVideoProvider"
+]
+```
+
+or
+
+```javascript
+[
+    "jwplayerBidAdapter",
+    "jwplayerRtdProvider"
+]
+```
+
+<a id="bid-request-parameters"></a>
+
+## Bid Params
+
+We support all oRTB params and encourage populating as many as possible.
+
+<a id="required-bidder-params"></a>
+
+### Required Bidder params
+
+You must include the following parameters at the bidder level.
```

Comment:
Since you just talked about ORTB params -- (I assume setConfig({ortb2}), this could be a little confusing. Maybe instead of "at the bidder level", perhaps "in the AdUnit bids array" would be more precise?

Reply:
I kept the wording because it's used other bid adapters (appnexus, pubmatic) but I clarified the property path to set it. Let me know if you disagree with this change